pydy/system.py

```diff
@@ -462,18 +463,37 @@ def generate_ode_function(self, generator='lambdify', **kwargs):
 
         """
 
-        if 'specified' in kwargs:
-            kwargs.pop('specified')
-            print("User supplied 'specified' kwarg was disregarded.")
+        try:
```

Comment:
Why are you putting everything in a `try-except` block? You can simply check if "Specified" is in kwargs, then issue a warning, and move ahead.